cmd/webui/ach/ach_js.go

```diff
@@ -24,6 +25,14 @@ func parseACH(input string) (string, error) {
 	return buf.String(), nil
 }
 
+func parseReadable(file *ach.File) (string, error) {
+	var buf bytes.Buffer
+	// passing nil for the masking Opts to be
+	// consistent with the JSON output that is not masked
+	describe.File(&buf, file, nil)
```

Comment:
nit: I recommend being explicit with the Opts and I think we could remove the code comments here
```suggestion
	describe.File(&buf, file,  &describe.Opts{
	                         MaskNames: false,
				MaskAccountNumbers:  false,
			})
```

Reply:
Either or. That package is designed to handle `nil` without issue.